README.md

```diff
@@ -30,6 +30,12 @@ dyrector.io helps engineer & DevOps teams and individual developers to shift the
 
 Join our Discord and connect with other members to share and learn together. If you like the project, give us a Star.
 
+## Getting started
+You have various options to set up the platform right away:
+- [CLI](./GETTING_STARTED.md) 
+- [SaaS](https://app.dyrectorio.com) 
```

Comment:
We should indicate that this is ready to use (pre-alpha) version.

Reply:
We have a SaaS section later on the page where this is explained, should we use an anchor instead of this link then?

Reply:
Ofc.